loopy/ipython_ext.py

```diff
@@ -9,7 +9,7 @@
 class LoopyMagics(Magics):
     @cell_magic
     def fortran_kernel(self, line, cell):
-        result = lp.parse_fortran(cell)
+        result = lp.parse_fortran(cell, return_list_of_knls=True)
```

Comment:
At some places, for ex. where we fuse kernels post fortran parsing, we cannot return a `Program` because we don't have a "root kernel". I am open to allowing it to fail until we have the logic of `TranslationUnit` in kcv3.

Reply:
Let's let it fail until we no longer need root kernels.

Reply:
Done in 266fea.